### ISSUE TITLE
Page became unresponsive after clicking the go back button

### ISSUE THREAD

**Issue by OneMoreJack** [antfu-collective/icones#306]
### Describe the bug

Everytime I clicked the go back button on the top-left corner, the page became unresponsive.

Reproduction Steps:
1. open https://icones.js.org/ in your browser(Chrome in my case)
2. click an icon set
3. click the go back button 

<img width="1373" alt="image" src="https://github.com/antfu/icones/assets/45257542/10e3d1b9-0bdd-472b-adb0-ffc81881dcf6">

### Reproduction

https://icones.js.org/

### System Info

```Shell
System:
    OS: macOS 14.2.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 169.36 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 16.18.1 - ~/.nvm/versions/node/v16.18.1/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 9.6.0 - ~/.nvm/versions/node/v16.18.1/bin/npm
    pnpm: 8.15.6 - ~/Library/pnpm/pnpm
  Browsers:
    Chrome: 123.0.6312.122
    Safari: 17.2.1
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by antfu** [comment]
There was a big and get fixed a few days ago. Can you clean the cache and make sure you are on the latest version and see if it still presist? Thanks

**Comment by OneMoreJack** [comment]
I cleaned the cache in my browser, checked the "Disable Cache" checkbox in devtools, and reload the page, it turned out that the problem was still there.

<img width="1725" alt="Snipaste_2024-04-12_23-36-09" src="https://github.com/antfu/icones/assets/45257542/dc8ccdb3-7d25-4ac1-aa33-7affaa6ea951">


Then, I thought that may be, it was the "Service Worker" caused the issue, therefore, I manually unregistered it. 
Now I'm facing another problem 😂, Everytime I reload  "https://icones.js.org/": 
1. the site will verify if I'm a real human
2. then there comes some 403 error in networks and the page remain empty

<img width="1723" alt="image" src="https://github.com/antfu/icones/assets/45257542/e0297fa9-a699-489c-9d43-36a1ff6e35bb">

I did some research and run the following code in Console
```js
navigator.serviceWorker.register("/serviceworker.js");
```
then I got a 403 error.

What can I do to register the service work back?
